modules/builtin/src/content-types/carousel.js

```diff
@@ -193,7 +132,7 @@ module.exports = {
       ...base.typingIndicators
     }
   },
-  newSchema:{
+  newSchema: {
```

Comment:
There was jsonSchema and uiSchema, so we used this to differentiate them (instead of just adding "schema"). There's a couple of stuff that will be refactored & fixed before the release. Since it will only be renaming stuff around, this can be done in its own PR

Reply:
ok, agreed on that